castle/cms/browser/controlpanel/sessions.py

```diff
@@ -10,15 +10,22 @@ class SessionsView(BrowserView):
     description = "View and control your site's currently logged-in user sessions."
 
     def __call__(self):
+        if self.request.REQUEST_METHOD == 'POST':
+            if self.request.form.get('id'):
+                session_id = self.request.form.get('id')
+                user_id = self.request.form.get('userid')
+                user = api.user.get(user_id)
+                sm = SessionManager(self.context, self.request, user)
+                sm.session_id = session_id
+                sm.expire()
+            if self.request.form.get('timeout'):
```

Comment:
Wait, what is this for?

Reply:
This is the code that actually changes the timeout attribute of the session plugin if they've submitted a value.  Seemed like why have a separate configuration registry key or something when the only reason for it would be to set the plugin's timeout to it's value.

Reply:
yah, we don't want a separate registry key but I think it should be configured in the security control panel. It can still modify that value though.